### PR TITLE
Surface failed fetches in the performance timeline

### LINKED ISSUE #12: Clarify presence of requests that don't return a response
https://lists.w3.org/Archives/Public/public-web-perf/2015Feb/0065.html

> There's some differences in the way browsers treat requests that don't
return a response. FF Nightly and IE 11 both create PerformanceResourceTiming entries for
these requests, whereas Chrome & Canary don't. The RT spec isn't explicit as to whether requests that miss a response should be included. -- @andydavies

### PR DESCRIPTION
**Pull request by igrigorik**
- fetches that are blocked (CSP, CORS, etc) are omitted
- fetches aborted due to network/other errors must be included
- failed fetches must surface initialized attributes up to point of
  failure

Closes #12.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -257,12 +257,9 @@
...
+
```
Comment (yoavweiss) [inline index.html]: Can we avoid extremely long lines in the future? It makes it so much harder to review changes.

**Comment by yoavweiss** [comment]
LGTM, but interested in @mikewest's opinion as well.

File: index.html (hunk 1 above)
Comment (mikewest) [inline index.html]: 80 columns FTW!

**Comment by mikewest** [comment]
How do you deal with redirects? That is, do you expose the endpoints to which redirection directs, or do you tie everything to the original url that the UA fetched?

**Comment by igrigorik** [comment]
@mikewest currently, we don't expose redirection end-points (see [step in 19](https://rawgit.com/w3c/resource-timing/0eb0f6997fc3f8a70a556212b45fa9ce5cfe7631/index.html#process) for full details). 

That said, we do have an open discussion to change this, see: https://lists.w3.org/Archives/Public/public-web-perf/2015Feb/0059.html

**Comment by toddreifsteck** [comment]
LGTM

**Comment by igrigorik** [comment]
Re, redirects, see: https://github.com/w3c/resource-timing/issues/21

File: index.html [hunk 2]
```diff
@@ -257,12 +257,9 @@
 
 <section id="resources-included">
```
Comment (Safinssssss) [inline index.html]: king

File: index.html (hunk 1 above)
Comment (Safinssssss) [inline index.html]: king

**Comment by eligrey** [comment]
Was Chrome ever updated to reflect this spec change?

Screen Shot 2020-05-04 at 12 38 44 PM

**Comment by npm1** [comment]
For 404 at least it was supposed to be fixed, per https://bugs.chromium.org/p/chromium/issues/detail?id=883400. Please comment on that issue if you're not seeing that fixed in a recent Chrome version.

**Pull request merged**